### ISSUE TITLE
Don't have nuts

### ISSUE THREAD

**Issue by hasecilu** [bilbaomakers/StemfieWB#3]
The STEMFIE system consists of several components as fasteners, shafts, gears, etc.

I think it's beneficial to have more components available on the workbench.

<https://get.stemfie.org/>

---

## Beams

- [X] Beams - Angled - T-shaped Symmetric - Square Ends
- [X] Beams - Angled - U-shaped Symmetric - Square Ends
- [X] Beams - Straight - Box-Section - Square Ends/
- [X] Beams - Straight - Standard - Square Ends
- [ ] #16

## Braces

- [X] Braces - Angled - 90 Degrees - Asymmetric - Round Ends
- [X] Braces - Angled - 90 Degrees - Symmetric - Round Ends
- [X] Braces - Cross - Standard - Symmetric - Round Ends
- [X] Braces - Straight - Slotted - Both Ends - Round Ends
- [X] Braces - Straight - Slotted - Centered - Round Ends
- [X] Braces - Straight - Slotted - Full Length - Round Ends
- [x] Braces - Straight - Slotted - Sequential - Round Ends
- [X] Braces - Straight - Slotted - Single End - Round Ends
- [X] Braces - Straight - Standard - Round Ends

## Connectors

- [X] 1 way
- [X] 2 ways 90°
- [X] 2 ways 180°
- [X] 3 ways
- [X] 4 ways

#### Cam-locking

- [ ] Open - <https://get.stemfie.org/Fasteners%20-%20Nuts%20-%20Cam-Locking%20-%20Open/>
#### Threaded

- [ ] Barrel through hole center - <https://get.stemfie.org/Fasteners%20-%20Nuts%20-%20Threaded%20-%20Barrel%20-%20Symmetric%20-%20Through-Hole%20Center%20-%20Threaded%20Double-Ended/>
- [ ] Open - <https://get.stemfie.org/Fasteners%20-%20Nuts%20-%20Threaded%20-%20Open/>
- [ ] Barrel threaded center - <https://get.stemfie.org/Fasteners%20-%20Nuts%20-%20Threaded%20-%20Barrel%20-%20Symmetric%20-%20Threaded%20Center-Double-Ended/>
- [ ] Cap nuts - <https://get.stemfie.org/Fasteners%20-%20Nuts%20-%20Threaded%20-
%20Cap%20Nuts/>
- [ ] Adhesive mount - <https://get.stemfie.org/Fasteners%20-%20Nuts%20-%20Threaded%20-%20Adhesive-Mount/>

### Pins

- [ ] Spring pins locating unidirectional - <https://get.stemfie.org/Fasteners%20-%20Pins%20-%20Spring%20Pin%20-%20Locating%20-%20Asymmetric%20-%20Unidirectional/>
- [ ] Spring pin inner - <https://get.stemfie.org/Fasteners%20-%20Pins%20-%20Spring%20Pin%20-%20Inner/>
- [ ] Cam locking - <https://get.stemfie.org/Fasteners%20-%20Pins%20-%20Cam-Locking%20-%20Rounded%20Head/>
- [ ] Spring pin locating bidirectional - <https://get.stemfie.org/Fasteners%20-%20Pins%20-%20Spring%20Pin%20-%20Locating%20-%20Symmetric%20-%20Bidirectional/>

### Screws

- [ ] Fully threaded RHD- <https://get.stemfie.org/Fasteners%20-%20Screws%20-%20Fully%20Threaded%20-%20Rounded%20Head/>
- [ ] Shoulder screws RHD - <https://get.stemfie.org/Fasteners%20-%20Shoulder%20Screws%20-%20Rounded%20Head/>
- [ ] Fully threaded STEMFIE HEAD - <https://get.stemfie.org/Fasteners%20-%20Screws%20-%20Fully%20Threaded%20-%20STEMFIE%20Head/>

### Spacers
- [x] Bud - Free - <https://get.stemfie.org/Fasteners%20-%20Spacers%20-%20BUD%20-%20Free/>
- [x] Fixed - <https://get.stemfie.org/Fasteners%20-%20Spacers%20-%20Fixed/>
- [x] Free - <https://get.stemfie.org/Fasteners%20-%20Spacers%20-%20Free/>
- [ ] Free - Adhesive mount <https://get.stemfie.org/Fasteners%20-%20Spacers%20-%20Free%20-%20Adhesive-Mount/>

### Threaded rods
- [ ] Fully threaded - <https://get.stemfie.org/Fasteners%20-%20Threaded%20Rods%20-%20Fully%20Threaded/>

### Washers
- [ ] #18
- [ ] #17

## Plates

- [X] Triangular plate
- [X] Square plate - in reality we have rectangular plate
- [X] Hexagonal plate 

## Shafts

- [X] Plain - <https://get.stemfie.org/Shafts%20-%20Plain/>
- [X] Indexing - <https://get.stemfie.org/Shafts%20-%20Indexing/>

## Springs

- [ ] Rotor - <https://get.stemfie.org/Springs%20-%20Rotor/>

## Tools

- [ ] Driver Nut - https://www.stemfie.org/parts?view=article&id=161:driver-nut-spn-tol-0034&catid=40
- [ ] Driver Screw Nut - https://www.stemfie.org/parts?view=article&id=159:driver-screw-nut-spn-tol-0001&catid=40
- [ ] Driver Nut Screw RHD - https://www.stemfie.org/parts?view=article&id=181:spn-tol-0035&catid=40
- [ ] Spanner Nut - https://www.stemfie.org/parts?view=article&id=160:spanner-nut-spn-tol-0004&catid=40

**Comment by Stemfie3D** [comment]
I fully agree. The complete set of CAD files for adding new parts to the workbench can be fetched from get.stemfie.org.

**Comment by hasecilu** [comment]
If there is no response I could step as maintainer in my fork... ideally parts would be parametrically created, but for the time being could make a command to download from cloud, any help is appreciated.
